pyaedt/desktop.py

```diff
@@ -103,30 +101,24 @@ def update_aedt_registry(key, value, desktop_version="193"):
     desktop_version : str, optional
         Version of AEDT to use. The default is ``"193"`` 
```

Comment:
@jleonatti the default version is 193 while in the conftest.py file, we use the 2021.1 as parameter of the Desktop class
`desktop = Desktop("2021.1", NG=True)`.

Is it expected?

Reply:
@MaxJPRey that default version of 193 is only for the `update_aedt_registry` method. I'm not sure why that specific version was chosen in that case. But in the case of starting AEDT through the Desktop object, you can specify the version you want to start in the Desktop constructor or it will select the newest version, I believe. I think that most users will be using newer version of AEDT here, which is why I selected 2021.1.

Reply:
@jleonatti, I think this should be raised as an issue.

Reply:
@akaszynski is the issue that 193 is the default version or that the format is expected to be "123" instead of "1234.5"? Or both?

Reply:
Issue is default version.  Do we even "officially" support 19.3 as of 2021R2?